.github/workflows/event-push-to-integ.yml

```diff
@@ -7,6 +7,12 @@ on:
     branches:
       - master
       - '[0-9]+.[0-9]+'
+    paths: # Only run on changes to these paths (code, deps, and benchmarks related changes)
```

Comment:
Is that too permissive? what if you change flow tests only? install files? other github actions flows?